common/src/main/java/org/terracotta/angela/common/distribution/Distribution107InlineController.java

```diff
@@ -309,15 +309,20 @@ private List<String> addOptions(TerracottaServer server, File workingDir) {
     }
 
     if (server.getSecurityDir() != null) {
-      throw new IllegalStateException("this option is broken for angela: FIX");
+      options.add("-x");
+      Path securityRootDirectoryPath = workingDir.toPath().resolve("security-root-directory-" + server.getServerSymbolicName().getSymbolicName());
+      server.getSecurityDir().createSecurityRootDirectory(securityRootDirectoryPath);
+      options.add(securityRootDirectoryPath.toString());
     }
 
     if (server.isSslTls()) {
       options.add("-t");
+      options.add("true");
```

Comment:
I am not sure this is required because just the option being present means true if I remember

Reply:
it is required my server startup is failing without that u can't just pass the option u have the pass the value as well.